### PR TITLE
Add input source for Shesmu actions

### PR DESCRIPTION
**Pull request by apmasell**
This is the crazy self-referential thing Morgan proposed.

### CODE REVIEW

File: shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionValue.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package ca.on.oicr.gsi.shesmu;
+
+import java.time.Instant;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ShesmuIntropsectionValue {
```
Comment (iainrb) [inline shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionValue.java]: Typo - "Intropsection"

File: shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionValue.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	private final Action action;
+	private final Instant changed;
+	private final Instant checked;
+	private final Instant generated;
+	private final Set<Tuple> locations;
+	private final ActionState state;
+
+	public ShesmuIntropsectionValue(Action action, Instant changed, Instant checked, Instant generated, ActionState state,
```
Comment (iainrb) [inline shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionValue.java]: Typo

File: shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionRepository.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package ca.on.oicr.gsi.shesmu;
+
+public interface ShesmuIntropsectionRepository extends InputRepository<ShesmuIntropsectionValue> {
```
Comment (iainrb) [inline shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionRepository.java]: Typo - "Intropsection"

File: shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionProcessorRepository.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package ca.on.oicr.gsi.shesmu;
+
+import java.util.Map;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import org.kohsuke.MetaInfServices;
+
+@MetaInfServices
+public class ShesmuIntropsectionProcessorRepository implements ShesmuIntropsectionRepository {
```
Comment (iainrb) [inline shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/ShesmuIntropsectionProcessorRepository.java]: Typo

**Comment by apmasell** [comment]
No, I'm just terrible as spelling.

**Pull request merged**